parts/common/components.json

```diff
@@ -1263,8 +1263,8 @@
           "current": {
             "versionsV2": [
               {
-                "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.32.0"
+                "renovateTag": "registry=https://packages.microsoft.com/azurelinux/3.0/prod/cloud-native/x86_64/repodata/, name=kubernetes-cri-tools, os=azurelinux, release=3.0",
```

Comment:
Shouldn't this need to be under a azurelinux field ?

Why don't we have a mariner field ?

Reply:
I was simply looking at how we do it for containerD and saw it was seperated.  Also know mariner2 is EOL. They will probably stop updating new version in v2 and add more in v3